Src/ILGPU/Context.cs

```diff
@@ -384,6 +385,70 @@ public DeviceCollection<TDevice> GetDevices<TDevice>()
                 : new DeviceCollection<TDevice>(new List<Device>());
         }
 
+        /// <summary>
+        /// Attempts to return the most optimal single device.
+        /// </summary>
+        /// <param name="preferCPU">Always returns CPU device 0.</param>
+        /// <returns>Selected device.</returns>
+        public Device GetPreferredDevice(bool preferCPU)
+        {
+            if (preferCPU)
+            {
+                return GetDevice<CPUDevice>(0);
+            }
```

Comment:
MINOR: The style convention in ILGPU is for bracketless single-line `if` statements.
```suggestion
            if (preferCPU)
                return GetDevice<CPUDevice>(0);
```